src/main/java/frc/robot/commands/ShootCommand.java

```diff
@@ -74,7 +74,7 @@ public void execute() {
     var distanceToSpeaker = robotTranslation.getDistance(speakerTranslation);
 
     // Lookup shooter settings for this distance
-    var shootingSettings = SHOOTER_INTERPOLATOR.calculate(distanceToSpeaker);
+    var shootingSettings = SHOOTER_PITCH_INTERPOLATOR.calculate(distanceToSpeaker);
```

Comment:
Why is this changed? The interpolator does both pitch and velocity, why put only pitch in the name?